src/cmd/singularity/cli_test.go

```diff
@@ -44,6 +46,17 @@ func run(m *testing.M) int {
 		log.Fatalf("singularity config is not a regular file")
 	}
 
+	// Make temp dir for tests
+	name, err := ioutil.TempDir("", "stest.")
+	if err != nil {
+		log.Fatalf("failed to create temporary directory: %v", err)
+	}
+	defer os.RemoveAll(name)
+	if err := os.Chmod(name, 0777); err != nil {
```

Comment:
Is the mode pre-umask or is umask ignored?

Did we have this conversation already? It seems like we might have had this conversation already...

Reply:
I believe the `ioutil.TempDir` will take `umask` into account... but the `os.Chmod` will not.